Makefile.common

```diff
@@ -0,0 +1,54 @@
+INCFLAGS := -I$(CORE_DIR) -I$(CORE_DIR)/arm
+
+SOURCES_C :=   $(CORE_DIR)/arm/arm.c \
+					$(CORE_DIR)/arm/isa-thumb.c \
+					$(CORE_DIR)/arm/isa-arm.c \
+					$(CORE_DIR)/arm/decoder-arm.c \
+					$(CORE_DIR)/arm/decoder-thumb.c \
+					$(CORE_DIR)/arm/decoder.c \
+					$(CORE_DIR)/debugger/debugger.c \
+					$(CORE_DIR)/debugger/memory-debugger.c \
+					$(CORE_DIR)/gba/savedata.c \
+					$(CORE_DIR)/gba/io.c \
+					$(CORE_DIR)/gba/rr/vbm.c \
+					$(CORE_DIR)/gba/rr/mgm.c \
+					$(CORE_DIR)/gba/bios.c \
+					$(CORE_DIR)/gba/hle-bios.c \
+					$(CORE_DIR)/gba/input.c \
+					$(CORE_DIR)/gba/audio.c \
+					$(CORE_DIR)/gba/memory.c \
+					$(CORE_DIR)/gba/cheats.c \
+					$(CORE_DIR)/gba/cheats/gameshark.c \
+					$(CORE_DIR)/gba/cheats/parv3.c \
+					$(CORE_DIR)/gba/cheats/codebreaker.c \
+					$(CORE_DIR)/gba/gba.c \
+					$(CORE_DIR)/gba/hardware.c \
+					$(CORE_DIR)/gba/sio.c \
+					$(CORE_DIR)/gba/sio/lockstep.c \
+					$(CORE_DIR)/gba/video.c \
+					$(CORE_DIR)/gba/serialize.c \
+					$(CORE_DIR)/gba/supervisor/rr.c \
+					$(CORE_DIR)/gba/supervisor/cli.c \
```

Comment:
Are cli.c and rr.c actually necessary? I need to make it possible to compile without those two files if that's not the case.